tokio/src/fs/write.rs

```diff
@@ -19,7 +19,7 @@ use std::{io, path::Path};
 /// # Ok(())
 /// # }
 /// ```
-pub async fn write<C: AsRef<[u8]> + Unpin>(path: impl AsRef<Path>, contents: C) -> io::Result<()> {
+pub async fn write(path: impl AsRef<Path>, contents: impl AsRef<[u8]>) -> io::Result<()> {
```

Comment:
Why was this even `+ Unpin` in the first place?

Reply:
No idea... probably accident / left over.